src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/ShipmentUnit.xml

```diff
@@ -32,8 +32,6 @@
             <itemOperation name="get" />
         </itemOperations>
 
-        <collectionOperations />
```

Comment:
Why have you removed this?

Reply:
Because we have it inline 29 - i fixed mistake after merge :( (btw i will move it above `itemOperations` i next PR :) )